dbos/_queue.py

```diff
@@ -32,10 +32,12 @@ def __init__(
         self,
         name: str,
         concurrency: Optional[int] = None,
+        worker_concurrency: Optional[int] = None,
         limiter: Optional[QueueRateLimit] = None,
     ) -> None:
         self.name = name
         self.concurrency = concurrency
+        self.worker_concurrency = worker_concurrency
```

Comment:
We could raise an exception if `worker_concurrency > concurrency`, to help the user detect a (likely) erroneous configuration.